content/applications/sales/point_of_sale/restaurant/floors_tables.rst

```diff
@@ -2,91 +2,97 @@
-to your main interface.
+Add a floor by clicking :guilabel:`+ Add Floor`, which opens a pop-up window where you can choose a
+name for it.
 
-.. image:: floors_tables/restaurant_07.png
-    :align: center
-    :alt: View of the pos interface to register orders
+Once a floor is created, add a table by clicking :guilabel:`+ TABLE`. You can move it by selecting
+it and drag and dropping it as desired. Once a table is selected, you can also change the number of
+seats available by clicking :guilabel:`SEATS`, the table shape by clicking :guilabel:`SHAPE`, the
+table color by clicking :guilabel:`FILL`, or the table name by clicking :guilabel:`RENAME`. To
+duplicate an existing table, select it and click :guilabel:`COPY`. You can also remove a table by
+clicking :guilabel:`DELETE`.
 
-Transfer customer(s)
-====================
+Click :guilabel:`CLOSE` once you are satisfied with your floor plan to save it.
 
-If your customers want to move to another table after they already
-ordered, use the transfer button. This way, the order is also moved to the new
-table.
+.. image:: floors_tables/floor-map.png
 
-To do so, select the table your customer is currently on.
+.. note::
+   If no table is selected, the modification are applied to the floor.
 
-.. image:: floors_tables/restaurant_08.png
-    :align: center
-    :alt: View of the restaurant tables, one having a pending order
+.. warning::
+   Removing a table or a floor cannot be undone.
 
-Now, click on the transfer button and select the table to which you are transferring your
-customer.
+.. _floors_tables/orders:
 
-.. image:: floors_tables/restaurant_09.png
-    :align: center
-    :alt: View of the pos interface and transfer button. How to transfer customers from one table
-          to another
+Take orders
+===========
 
-Register an additional order
-============================
+Click a table to access the POS interface and start taking your customer's order. The system
+automatically associates the order to the chosen table, allowing you to add more items and generate
+a bill specifically tied to the orders placed at that table.
 
-When registering an order, use the + button to simultaneously proceed to
-another one.
+Once an order is placed, go back to the floor plan view by clicking :guilabel:`BACK` and select
+another table to register another order.
 
-Then, you can shift between your orders and process the payment
-when needed.
+.. note::
+   As soon as you click a table, the number of guests is automatically set to one guest. If you
+   select a table by mistake, click :guilabel:`Release table` to free the selected table, or
+   :ref:`transfer the customer <floors_tables/transfer>` to another table.
 
-.. image:: floors_tables/restaurant_10.png
-    :align: center
-    :alt: View of the - button, allowing employees to close/remove an order
+.. _floors_tables/transfer:
 
-.. tip::
-   The - button allows you to remove the order you are currently on.
+Table transfer
+==============
```

Comment:
I would add a screenshot in this section, I managed to find it pretty easily but still can be nice to specify in which pos view it  can be found.